### DISCUSSION TITLE
Groups, player icons missing

### DISCUSSION THREAD

**Discussion by wokkeltje13** [punxaphil/custom-sonos-card#565] [Q&A] [answered]
In the documentation I noticed that in the Groups tab, there is an image of the player in front of the name. In my setup, I don't see this icon.
What am I doing wrong?

**Comment by punxaphil** [comment, marked as answer]
Readme updated:
https://github.com/punxaphil/custom-sonos-card?tab=readme-ov-file#device-icons


---
PunxaPhil
[![image](https://github.com/user-attachments/assets/cb2e240e-f9ca-4900-a425-b8089216c1ee)](https://patreon.com/PunxaPhil) [![image](https://github.com/johanfrick/custom-sonos-card/assets/26582898/9bebc1f6-6bda-4cbd-b596-50d3d2f712de)](https://www.buymeacoffee.com/punxaphil)
